packages/twenty-website/src/content/developers/self-hosting/setup.mdx

```diff
@@ -139,84 +139,86 @@ yarn command:prod cron:calendar:ongoing-stale
-    ['IS_MULTIWORKSPACE_ENABLED', 'false', 'Allows the use of multiple workspaces. Requires a web server that can manage wildcards for subdomains.'],
-    ['PASSWORD_RESET_TOKEN_EXPIRES_IN', '5m', 'Password reset token expiration time'],
-    ]}></ArticleTable>
+### Required Infrastructure
+```bash
+# Database & Cache
+PG_DATABASE_URL=postgres://user:password@localhost:5432/twenty
+REDIS_URL=redis://localhost:6379
 
-### Email
+# Server Configuration  
+SERVER_URL=http://localhost:3000
+APP_SECRET=your-secret-key-here
+
+# Frontend URL (defaults to SERVER_URL if not set)
+FRONTEND_URL=http://localhost:3001
+```
+
+### Environment-Only Variables
+Some infrastructure and system-level variables cannot be changed through the admin panel and must remain in your `.env` file. These are marked as `isEnvOnly: true` in the codebase for security and infrastructure reasons.
+
+**Key categories that remain environment-only:**
+- **Database & Cache connections** (`PG_DATABASE_URL`, `REDIS_URL`)
+- **Core server configuration** (`SERVER_URL`, `APP_SECRET`, `NODE_ENV`)
+- **Infrastructure settings** (database pool, logging drivers, SSL certificates)
+
+
+## Configuration Modes
+
+### Default Mode (Database Configuration Enabled)
+- `IS_CONFIG_VARIABLES_IN_DB_ENABLED=true` (default)
+- Admin panel changes **override** .env values
+- Most settings can be changed without touching .env files
+
+![Config Variables](/images/user-guide/setup/admin-panel-config-variables-2.png)
+
+### Environment-Only Mode
+- Set `IS_CONFIG_VARIABLES_IN_DB_ENABLED=false`
+- Admin panel shows current values and descriptions but can't be changed
+- All configuration must be managed through .env files
+
+
+## Frontend
 
 <ArticleTable options={[
-    ['IS_EMAIL_VERIFICATION_REQUIRED', 'false', 'If enabled, users must verify their email address before signing in. When true, users will receive a verification email after registration'],
-    ['EMAIL_VERIFICATION_TOKEN_EXPIRES_IN', '1h', 'How long email verification tokens remain valid before requiring a new verification email'],
-    ['EMAIL_FROM_ADDRESS', 'contact@yourdomain.com', 'Global email From: header used to send emails'],
-    ['EMAIL_FROM_NAME', 'John from YourDomain', 'Global name From: header used to send emails'],
-    ['EMAIL_SYSTEM_ADDRESS', 'system@yourdomain.com', 'Email address used as a destination to send internal system notification'],
-    ['EMAIL_DRIVER', 'logger', "Email driver: 'logger' (to log emails in console) or 'smtp'"],
-    ['EMAIL_SMTP_HOST', '', 'Email SMTP Host'],
-    ['EMAIL_SMTP_PORT', '', 'Email SMTP Port'],
-    ['EMAIL_SMTP_USER', '', 'Email SMTP User'],
-    ['EMAIL_SMTP_PASSWORD', '', 'Email SMTP Password'],
-  ]}></ArticleTable>
+    ['REACT_APP_SERVER_BASE_URL', 'http://localhost:3000', 'Url of backend server'],
+    ['GENERATE_SOURCEMAP', 'false', 'Generate source maps for debugging'],
```

Comment:
Why keep this? Not useful for self-hosting imo, this guide should be built from a self-hoster perspective